android-interop-testing/app/build.gradle

```diff
@@ -19,6 +20,36 @@ android {
     }
 }
 
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:3.0.0-alpha-2'
+    }
+    plugins {
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java:0.8.0-SNAPSHOT'
+        }
+    }
+    generateProtoTasks {
+        all().each { task ->
+            task.builtins {
+                javanano {
+                    // Options added to --javanano_out
+                    option 'ignore_services=true'
+                }
+
+                task.plugins {
```

Comment:
The nesting here is a bit strange. It seems that there should be another } before task.plugins.